packages/traveling-fastlane/actions/manage_ad_hoc_provisioning_profile.rb

```diff
@@ -1,28 +1,46 @@
 require 'spaceship'
 require 'json'
 require 'base64'
+require 'optparse'
 require 'set'
 require_relative 'funcs'
 
 ENV['SPACESHIP_AVOID_XCODE_API'] = '1'
 
+options = {}
+OptionParser.new do |opts|
```

Comment:
I'd slightly prefer not to do that, please also see:
https://github.com/expo/turtle/pull/88#discussion_r292352943